plugins/usbdmx/UsbDmxPlugin.cpp

```diff
@@ -107,6 +108,16 @@ bool UsbDmxPlugin::SetDefaultPreferences() {
 
   return true;
 }
+
+void UsbDmxPlugin::ConflictsWith(
+    std::set<ola_plugin_id>* conflicting_plugins) const {
+  if (EuroliteProFactory::IsEuroliteMk2Enabled(m_preferences)) {
+    conflicting_plugins->insert(OLA_PLUGIN_USBPRO);
```

Comment:
Nit, sort the list.

Reply:
It was sorted on the order they were in the enum, but I agree that alphabetically makes more sense here. Done.

Reply:
Ah okay, yeah I think this is more understandable elsewhere thanks.